infrastructure/template/api-op-vh-cancel-hearing-policy.xml

```diff
@@ -0,0 +1,216 @@
+                    <set-header name="Content-Type" exists-action="override">
+                        <value>application/json</value>
+                    </set-header>
+                    <set-body>@{
+                    return new JObject(
+                    new JProperty("message", context.Request.Headers.GetValueOrDefault("Destination-System","") + " destination doesn't support this functionality")
+                    ).ToString();
+                    }</set-body>
+                </return-response>
+            </otherwise>
+        </choose>
+    </inbound>
+    <backend>
+        <base />
+    </backend>
+    <outbound>
+        <base />
+        <choose>
+            <when condition="@(context.Response.StatusCode == 400)">
+                <cache-remove-value key="vhAuth" />
+                <return-response>
+                    <set-status code="400" />
+                    <set-header name="Content-Type">
+                        <value>application/json</value>
+                    </set-header>
+                    <set-body>@{
+                return new JObject(
+                new JProperty("message", "Bad Request.")
+                ).ToString();
+                }
+            </set-body>
+                </return-response>
+            </when>
+            <when condition="@(context.Response.StatusCode == 401)">
+                <cache-remove-value key="vhAuth" />
+                <send-request ignore-error="true" timeout="20" response-variable-name="snowAuth" mode="new">
+                    <set-url>https://#{keyVaultHost}#/secrets/hmi-servicenow-auth?api-version=7.0</set-url>
+                    <set-method>GET</set-method>
+                    <authentication-managed-identity resource="https://vault.azure.net" />
+                </send-request>
+                <send-request ignore-error="false" timeout="20" response-variable-name="serviceNow" mode="new">
+                    <set-url>https://#{snowHost}#/api/now/table/incident?sysparm_fields=number</set-url>
+                    <set-method>POST</set-method>
+                    <set-header name="Authorization" exists-action="override">
+                        <value>
+                            @("Basic " + ((IResponse)context.Variables["snowAuth"]).Body.As
+                            <JObject>()["value"].ToString())
+                        </value>
+                    </set-header>
+                    <set-body>@{
+                        return new JObject(
+                        new JProperty("assignment_group", "56b756774fbd368057db0b318110c7bd"),
+                        new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
+                        new JProperty("category", "Data Issue"),
+                        new JProperty("contact_type", "Alert"),
+                        new JProperty("description", "HMI - Update VH By ID: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
```

Comment:
Maybe change description to make it clear its the cancel endpoint

Reply:
fixed.